motionplan/planManager.go

```diff
@@ -355,7 +355,7 @@ func (pm *planManager) planParallelRRTMotion(
 		}
 	} else {
 		if maps == nil {
-			goalNode := &basicNode{q: make([]referenceframe.Input, len(pm.frame.DoF())), pose: goal}
+			goalNode := &basicNode{q: make([]referenceframe.Input, len(pm.frame.DoF())), pose: spatialmath.Compose(goal, flipPose)}
```

Comment:
I don't understand why we would seed the goal this way instead of just generating the tree normally and then flipping when we finish